po4a-update-templates.sh

```diff
@@ -52,9 +52,9 @@ do
     path="${dirname#$SRC_DIR/}"
 
     if [ "$dirname" = "$SRC_DIR" ]; then
-        potname=$basename.pot
+        potname="$basename".pot
     else
-        potname=$path/$basename.pot
+        potname="$path/$basename".pot
```

Comment:
I'd put the whole thing (including the trailing `.pot`) inside the double quotes, just feels tidier.

Reply:
Thanks, I've implemented your suggestions in #522